src/main/java/uk/gov/hmcts/juror/api/moj/controller/response/FailedToAttendListResponse.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.juror.api.moj.controller.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Builder
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Schema(description = "Response containing juror details for failed to attend list")
+public class FailedToAttendListResponse {
```

Comment:
Is this class needed?

Reply:
nope. you're spot on, we're using the other common dto (JurorDetailsDto) 😄